third_party/workspace.bzl

```diff
@@ -12,105 +12,96 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# TensorBoard external dependencies that can be loaded in WORKSPACE files.
+"""
+TensorBoard external dependencies that can be loaded in WORKSPACE files.
+"""
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:java.bzl", "java_import_external")
-load("@io_bazel_rules_closure//closure:defs.bzl", "filegroup_external")
-load("@io_bazel_rules_closure//closure:defs.bzl", "web_library_external")
-load("@io_bazel_rules_webtesting//web/internal:platform_http_file.bzl", "platform_http_file")
+load("@io_bazel_rules_webtesting//web/internal:platform_http_file.bzl", "platform_http_file")  # buildifier: disable=bzl-visibility
```

Comment:
In reaction to Buildifier's complaint:
```
third_party/workspace.bzl:21: bzl-visibility: Module "@io_bazel_rules_webtesting//web/internal:platform_http_file.bzl" can only be loaded from files located inside "//web", not from "//third_party/workspace.bzl". (https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#bzl-visibility)
```

Reply:
Do you know how/why bazel complains? I suspect it is because the path has "internal" or "private". If my hypothesis is correct, perhaps we should use this technique to hide all of our .bzls :)

Reply:
Yup, it does a pretty simple check that the path contains those keywords:
https://github.com/bazelbuild/buildtools/blob/master/warn/warn_visibility.go

We could hide ours too, but it only protects us if consumers run and respect `buildifier` warnings!